src/synapso_cli/commands/init.py

```
@@ -112,65 +160,28 @@ def _get_default_config() -> Dict[str, Any]:
     try:
-        config: GlobalConfig = get_config(config_file)
-        vector_store_path = config.vector_store.vector_db_path
-        vector_store_type = config.vector_store.vector_db_type
-        typer.echo(f"Initializing vector store at {vector_store_path}")
-        _initialize_sqlite_db(vector_store_path)
-        vector_store = DataStoreFactory.get_vector_store(vector_store_type)
-        vector_store.setup()
+        response = rest_client.system_init()
+    except SynapsoRestClientError as e:
+        typer.echo(f"Synapso REST client error: {e}", err=True)
+        raise typer.Exit(1)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)